src/main/java/com/example/yiseul/converter/MemberConverter.java

```diff
@@ -0,0 +1,54 @@
+package com.example.yiseul.converter;
+
+import com.example.yiseul.domain.Member;
+import com.example.yiseul.dto.member.MemberCreateRequestDto;
+import com.example.yiseul.dto.member.MemberCursorResponseDto;
+import com.example.yiseul.dto.member.MemberPageResponseDto;
+import com.example.yiseul.dto.member.MemberResponseDto;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class MemberConverter {
+
+    public static Member convertMember(MemberCreateRequestDto createRequestDto) {
+
+        return new Member(createRequestDto.name(), createRequestDto.age(), createRequestDto.hobby());
+    }
+
+    public static MemberResponseDto convertMemberResponseDto(Member member) {
+
+        return new MemberResponseDto(member.getId(),
+                member.getName(),
+                member.getAge(),
+                member.getHobby(),
+                member.getCreatedAt(),
+                member.getCreatedBy());
+    }
+
+    public static MemberPageResponseDto convertMemberPageResponseDto(Page<Member> page) {
+        List<MemberResponseDto> memberResponseDtos = page.getContent().stream()
+                .map(member -> convertMemberResponseDto(member))
+                .collect(Collectors.toList());
+
+        return new MemberPageResponseDto(
+                memberResponseDtos,
+                page.getNumber(),
+                page.getSize(),
+                page.getTotalPages(),
+                page.getTotalElements(),
+                page.isFirst(),
+                page.isLast());
+    }
+
+    public static MemberCursorResponseDto convertMemberCursorResponseDto(List<Member> members, Long nextCursorId) {
+        List<MemberResponseDto> memberResponseDtos = members.stream()
+                .map(member -> MemberConverter.convertMemberResponseDto(member))
```

Comment:
```suggestion
                .map(MemberConverter::convertMemberResponseDto)
```
p3; 

메서드 래퍼런스를 사용하면 좀 더 깔끔해질 것 같아요!

Reply:
메서드 레퍼런스를 하니까 확실히 코드가 더 깔끔하고 눈에 잘 읽히네요. 수정해서 올리겠습니다!!